content/product/product_management/index.md

```diff
@@ -16,25 +16,24 @@ This page contains information that is relevant for how to do well at your job a
 
-In addition to this shared content, each team has a set of sources of truth (that do not overlap) that they should update regularly:
+#### OKR and roadmap tracking
 
-#### Per-area strategy pages
-
-We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) to communicate where each of our product teams is headed, for the areas of the product that they work on. Newly created teams should begin documenting the scope of their team as well as their mission, vision, principles, and more by creating one of these pages as part of the process of forming. Having a page like this will help you ensure your own framework for prioritization is clear, and can also be a resource for those outside your team who want to understand why you are working on what you are.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your team to update your plans and status. After updating, you should ensure it is reflected in the manually created slide decks which rely on this information:
 
-These are updated monthly by each team, but wouldn't normally change too much each month. They are updated via a pull request, that anyone on the team can contribute to, and that can be shared with stakeholders as a summary of the change.
+- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker.
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
 
-#### PMM Roadmap slides
+#### Per-area strategy pages
 
-In addition to updating the strategy pages monthly, at the same time the team should also update the [PMM roadmap](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming important customer-facing features.
+We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a framework for each team's prioritization, to document the trade-offs between goals the teams are making, and to communicate where they are headed in the longer term not in terms of specific deliverables but in terms of strategy. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```

Comment:
```suggestion
We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) as a narrative framework for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed.
```